apps/right-to-rent-check/fields.js

```diff
@@ -52,5 +52,16 @@ module.exports = {
     mixin: 'input-text',
     validate: 'required',
     labelClassName: 'visuallyhidden'
+  },
+  'representative': {
+    mixin: 'radio-group',
+    validate: 'required',
+    legend: {
+      className: 'visuallyhidden'
```

Comment:
is this right, didn't you recently have a problem with this? I thought it was supposed to be `legenClassName: 'visuallyhidden'`.